apps/modernization-ui/src/settings.scss

```diff
@@ -47,9 +47,19 @@ td span {
 }
 
 .css-1jqq78o-placeholder {
-     color: $color-black !important;
+    color: $color-black !important;
 }
 
 .text-red {
     color: $color-red-50 !important;
 }
+
+.usa-pagination__button:active,
+.usa-pagination__button:focus,
+.usa-pagination__button:hover {
+    border: 1px solid #1a4480 !important;
+}
+
+.card-header-label {
+    font-size: 13px !important;
```

Comment:
issue:  this is defined in figma as 

```css
font-size: 0.8125rem;
font-weight: 600;
letter-spacing: 0.02031rem;
text-transform: uppercase; 
```
The color is specifed as #71767a which is defined in the `$color-gray-20` color variable.